java/fury-core/src/main/java/org/apache/fury/Fury.java

```diff
@@ -19,47 +19,18 @@
 
 package org.apache.fury;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.nio.ByteOrder;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.function.Consumer;
-import java.util.function.Function;
-import javax.annotation.concurrent.NotThreadSafe;
 import org.apache.fury.builder.JITContext;
 import org.apache.fury.collection.IdentityMap;
-import org.apache.fury.config.CompatibleMode;
-import org.apache.fury.config.Config;
-import org.apache.fury.config.FuryBuilder;
-import org.apache.fury.config.Language;
-import org.apache.fury.config.LongEncoding;
+import org.apache.fury.config.*;
```

Comment:
Fory don't use `*`  for imports, please execute `mvn spotless:apply & mvn checkstyle:check`